src/requirements/requirement-frontend-computation.ts

```diff
@@ -247,6 +248,15 @@ export default function computeGroupedRequirementFulfillmentReports(
         }
         break;
       }
+      case 'Grad': {
```

Comment:
Same comment here and throughout this file. Should this code more resemble the college code instead of the major/minor code?

Reply:
@willespencer Based on the implementation plan, it seemed that the idea was to treat the grad program as the same level as a major/minor since grad programs are technically under colleges. So I think this is ok? What are your thoughts, @willespencer and @vaishnavi17?

Reply:
That sounds good to me! Good to leave this as is.